papers/orhan_eroglu/00_geocat.rst

```diff
@@ -93,54 +93,52 @@ on the development of data analysis and visualization functions that operate
-for all of those tools.
+The software tools maintained by GeoCAT implement
+continuous integration and continuous delivery/deployment (CI/CD).
 
 Continuous Integration and Continuous Delivery/Deployment (CI/CD)
 =================================================================
 
-GeoCAT employs a continuous delivery model, which has been achieved through the use
-of a monthly package release cycle on package management systems and package indexes
+GeoCAT employs a continuous delivery model, with 
+a monthly package release cycle on package management systems and package indexes
 such as Conda :cite:`conda` and PyPI :cite:`conda`. This model helps the team make
 new functions available as soon as they are implemented and address potential errors
-pretty quickly. To assist this process, the team utilizes advanced  technologies
-throughout Github assets to ensure automation, unit testing and code coverage, as
+quickly. To assist this process, the team utilizes multiple tools
+throughout GitHub assets to ensure automation, unit testing and code coverage, as
 well as licensing and reproducibility. Figure :ref:`fig1badges`, for example,
 illustrates the set of badges from one of our software tools, GeoCAT's computational
 component, GeoCAT-comp, which shows the near real-time status of each CI/CD
-implementation at any time in the Github repository homepage.
+implementation at any time in the GitHub repository homepage.
 
 CI build tests of our repositories are implemented and
 automated (for pushed commits, pull requests, and daily scheduled execution) via
-Github Actions workflows :cite:`githubactions`, and the "CI" badge shows if those
-workflows are passing or failing. Similarly, the "CONDA-BUILDS" badge shows if the
+GitHub Actions workflows :cite:`githubactions`, and the "CI" badge displays the status of those workflows.
+Similarly, the "CONDA-BUILDS" badge shows if the
 conda recipe works successfully for the repository. The percent code coverage through
 unit tests are being analyzed with the "codecov" Python package :cite:`codecov`, and
 its overall results as well as details per code script can be seen via the "COVERAGE"
 badge. Each of our software repositories has a corresponding documentation page that
 is populated mostly-automatically through Sphinx Python documentation generator
 :cite:`sphinx` and published through ReadTheDocs :cite:`readthedocs` via an automated
-building/versioning schema. The "DOCS" badge provides a link to the documentation page
+building and versioning schema. The "DOCS" badge provides a link to the documentation page
 as well as shows failures, if any, with the documentation rendering process. Figure
 :ref:`fig2rtd` shows the documentation homepage of GeoCAT-comp for instance. The
 "NCAR" and "PYPI" badges in the "Package" row shows and links to the latest versions
 of the software tool distributed through NCAR's Conda channel and PyPI, respectively.
```

Comment:
Should this part be updated to reflect that they are now hosted on conda-forge?

Reply:
Yes, that would be good (We can submit the paper as is, then update this part and badge until it gets published)